tests/cart/addItemToCart.spec.js

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from '@playwright/test';
+const HomePageUrl = 'https://magento.softwaretestingboard.com/';
+
+test('Navigate to top tees', async ({ page }) => {
+  await page.goto(HomePageUrl);
+  await page.locator('#ui-id-5').hover();
+  await page.locator('#ui-id-17').hover();
+  await page.locator('#ui-id-21').click();
```

Comment:
getByTestId('ui-id05') then the same for the rest

Reply:
sounds great, doesn't work
![2023-12-12 08_32_56-Playwright Test](https://github.com/adrianmaciuc/break-it-playwright-javascript-project/assets/48152758/a265887d-64d9-43ff-b409-d5c1fc8bd34b)